### PR TITLE
IJPS-72

### PR DESCRIPTION
**Pull request by raakheerele18**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/IJPS-72

### Change description ###
Create a new page for Submit records to Finance (view)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/app/_workflows/submitter-workflow.service.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import { ViewSittingRecordService } from '../_services/sitting-records-service/view-sitting-records-service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SubmitterWorkflowService {
+
+  formData!: FormGroup;
+  userFormData!: FormGroup;
+  hasVisitedManage = false;
```
Comment (Josh-HMCTS) [inline src/app/_workflows/submitter-workflow.service.ts:15]: Consider changing this to something more like 'landingVisited' so its differentiated from other workflows

File: src/app/sitting-records/manage-sitting-records/manage-sitting-records.component.ts [hunk 2]
```diff
@@ -46,7 +46,7 @@ export class ManageSittingRecordsComponent implements OnInit {
     private cookies: CookieService,
     protected activatedRoute: ActivatedRoute,
     private formBuilder: FormBuilder,
-    private srWorkFlow: SittingRecordWorkflowService,
+    private srWorkFlow: RecorderWorkflowService,
```
Comment (Josh-HMCTS) [inline src/app/sitting-records/manage-sitting-records/manage-sitting-records.component.ts:49]: rename from srWorkFlow to match the new use of recorderWorkflow

File: src/app/sitting-records/submit-sitting-records/submit-sitting-records.component.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        </p>
+    
+        <ng-container style="width: 100%;">
+          <table 
+            datatable 
+            [dtOptions]="dtOptions" 
+            [dtTrigger]="dtTrigger"
+            class="govuk-table"
+            id="sittingRecordViewTable">
+            <caption></caption>
```
Comment (Josh-HMCTS) [inline src/app/sitting-records/submit-sitting-records/submit-sitting-records.component.html:27]: change from using <caption> to using aria-describedby= against table tag with description of table

File: src/app/_workflows/recorder-workflow.service.spec.ts [hunk 4]
```diff
@@ -1,25 +1,25 @@
 import { TestBed } from '@angular/core/testing';
 import { FormGroup, FormControl, FormBuilder } from '@angular/forms';
```
Comment (Josh-HMCTS) [inline src/app/_workflows/recorder-workflow.service.spec.ts:2]: Remove unused FormControl import

**Review by Josh-HMCTS — COMMENTED** [review]
.

File: src/app/_workflows/submitter-workflow.service.ts (hunk 1 above)
Comment (raakheerele18) [reply src/app/_workflows/submitter-workflow.service.ts]: Done

File: src/app/sitting-records/manage-sitting-records/manage-sitting-records.component.ts (hunk 2 above)
Comment (raakheerele18) [reply src/app/sitting-records/manage-sitting-records/manage-sitting-records.component.ts]: Done

File: src/app/sitting-records/submit-sitting-records/submit-sitting-records.component.html (hunk 3 above)
Comment (raakheerele18) [reply src/app/sitting-records/submit-sitting-records/submit-sitting-records.component.html]: Done

File: src/app/_workflows/recorder-workflow.service.spec.ts (hunk 4 above)
Comment (raakheerele18) [reply src/app/_workflows/recorder-workflow.service.spec.ts]: Done

**Review by Josh-HMCTS — APPROVED** [review]

**Pull request merged**